autoemulate/experimental/emulators/polynomials.py

```diff
@@ -20,6 +21,8 @@ def __init__(  # noqa: PLR0913
         self,
         x: TensorLike,
         y: TensorLike,
+        standardize_x: bool = False,
+        standardize_y: bool = False,
```

Comment:
Is it worth setting the default here to True (at least for the y) like with the MLP?

Reply:
From discussing, we think that having the defaults as `False` remains worthwhile so that users can opt-in to standardization and search over it in the model comparison loop still (e.g. `x_transforms_list=[[], [StandardizeTransform()]]`)